dashboard/app/models/script.rb

```diff
@@ -1296,7 +1296,9 @@ def summarize(include_lessons = true, user = nil, include_bonus_levels = false)
       hasStandards: has_standards_associations?,
       tts: tts?,
       is_course: is_course?,
-      background: background
+      background: background,
+      updatedAt: updated_at,
+      scriptPath: script_path(id: id)
```

Comment:
this resolves to something like http://localhost-studio.code.org:3000/s/653, which then redirects to something like http://localhost-studio.code.org:3000/s/valentine . I think you want `script_path(self)`, which gives something like /s/valentine . since the end result is the same, this could be ok to fix in a follow-up PR, so you can get this one merged sooner.

A bit of context on why we use the `lesson_path(id: id)` syntax in this PR: https://github.com/code-dot-org/code-dot-org/pull/37345